python/coinbase-agentkit/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ## [0.1.2] - 2025-02-14
 
 - Added gas configuration parameters (`gas_limit_multiplier`, `fee_per_gas_multiplier`) to `CdpWalletProvider` and `EthAccountWalletProvider`.
+- Fixed Morpho `deposit` and `withdraw` function headers to conform to the Action Provider Paradigm.
```

Comment:
please move this to the unreleased section of the CHANGELOG.md

Reply:
Great, I just made the update, thank @John-peterson-coinbase!